dagger-functions/src/test/java/io/odpf/dagger/functions/udfs/python/PythonUdfConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package io.odpf.dagger.functions.udfs.python;
+
+import io.odpf.dagger.common.configuration.Configuration;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static io.odpf.dagger.functions.common.Constants.*;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class PythonUdfConfigTest {
+
+    @Mock
+    private Configuration configuration;
+
+    @Before
+    public void setup() {
+        initMocks(this);
+    }
+
+    @Test
+    public void shouldParseConfig() {
+        String pythonJsonConfig = "{ \"PYTHON_FILES\": \"/path/to/function.zip\", \"PYTHON_ARCHIVES\": \"/path/to/file.txt\", \"PYTHON_REQUIREMENTS\": \"requirements.txt\", \"PYTHON_FN_EXECUTION_ARROW_BATCH_SIZE\": \"10000\", \"PYTHON_FN_EXECUTION_BUNDLE_SIZE\": \"100000\", \"PYTHON_FN_EXECUTION_BUNDLE_TIME\": \"1000\" }";
+
+        when(configuration.getString(PYTHON_UDF_CONFIG, "")).thenReturn(pythonJsonConfig);
+        PythonUdfConfig pythonUdfConfig = PythonUdfConfig.parse(configuration);
+
+        Assert.assertNotNull(pythonUdfConfig);
```

Comment:
can you check for other fields as well? one happy path

Reply:
ok